### PR TITLE
Add missing set_block_applied_callback

### PR DESCRIPTION
**Pull request by youaresofunny**
set_block_applied_callback works after patch

### CODE REVIEW

**Comment by dls-cipher** [comment]
@sschiessl-bcp please review

**Comment by clockworkgr** [comment]
> @sschiessl-bcp please review

@svk31  maintains this

**Comment by sschiessl-bcp** [comment]
`set_block_applied_callback` is defined to be supported in the readme, merging

**Comment by bogvas194-wq** [comment]
TFFAMQLZybALaLb4uxHA9RBE7pxhUAjF3U

File: lib/src/ChainWebSocket.js
Comment (bogvas194-wq) [inline lib/src/ChainWebSocket.js:245]: TJhWr7ofCp9DovyxzZFMZcD2yW1YkC7BGr

Comment (bogvas194-wq) [reply lib/src/ChainWebSocket.js:245]: TLqFHKbdvBjjmWt1RQW6DXcDPbWFcx2Cdp

Comment (bogvas194-wq) [reply lib/src/ChainWebSocket.js:245]: TLqFHKbdvBjjmWt1RQW6DXcDPbWFcx2Cdp

**Pull request merged**